docs/LSPSupport.md

```diff
@@ -323,12 +323,21 @@ the `codeInsight.parameterInfo` extension point. By default, LSP4IJ registers th
         implementationClass="com.redhat.devtools.lsp4ij.features.signatureHelp.LSPParameterInfoHandler"/>
 
 <codeInsight.parameterInfo
-id="LSPParameterInfoHandlerForTextMate"
-language="textmate"
-implementationClass="com.redhat.devtools.lsp4ij.features.signatureHelp.LSPParameterInfoHandler"/>
+        id="LSPParameterInfoHandlerForTextMate"
+        language="textmate"
+        implementationClass="com.redhat.devtools.lsp4ij.features.signatureHelp.LSPParameterInfoHandler"/>
 ```
 
 If you use another language, you will have to declare `codeInsight.parameterInfo` with your language.
+To override the IDE's default behaviour, use `order="first"`:

```

Comment:
I find using  `IDE's default behaviour` is hard to understand since the problem comes from when you install a plugin and it already defines the codeInsight.parameterInfo  for the language.

I would say a sentence like:

`if there is a plugin that already defines the `codeInsight.parameterInfo` for the language and you want to override it, you must use `order="first"``